druid-lookups/pom.xml

```diff
@@ -178,6 +178,12 @@
             <version>${druid.version}</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
check if is needed

Reply:
Removing from lookups, bumping version & fixing in core.